TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift

```diff
@@ -130,3 +140,12 @@ extension SecureStorage: SecureStorageProvider {
     }
   }
 }
+
+extension SecureStorageProvider {
+  public func get(
+    _ key: String,
+    attempts: Int? = nil
+  ) throws -> Data {
+    try get(key, attempts: attempts)
```

Comment:
We need a retry for the default implementation

Reply:
Yes this extension avoid any breaking change for current implementations, by making the attemps optonal, it will use the default number of attemps.